integration/benchmark/cmd/manual/main.go

```diff
@@ -43,7 +43,7 @@ func main() {
 	pushgateway := flag.String("pushgateway", "127.0.0.1:9091", "host:port for pushgateway")
 	profilerEnabled := flag.Bool("profiler-enabled", false, "whether to enable the auto-profiler")
 	_ = flag.Bool("track-txs", false, "deprecated")
-	accountMultiplierFlag := flag.Int("account-multiplier", 50, "number of accounts to create per load tps")
+	accountMultiplierFlag := flag.Int("account-multiplier", 100, "number of accounts to create per load tps")
```

Comment:
This one a one-time increase since we now wait around twice as long (verification takes time.) We can speed it up by:
* reusing setup state between runs. (market place simulator already does that).  (Can start factoring it out after #3091 lands.)
* not waiting for seals of account creating is also an option: this would at least make this phase faster.

As for two loaders pointing to the same benchmark cluster-- this may be outside of the scope right now: we are quite far from needing multiple concurrent benchmarks.